core-bundle/src/Resources/contao/library/Contao/Automator.php

```diff
@@ -471,6 +471,12 @@ public function rotateLogs()
 
 		foreach ($arrFiles as $strFile)
 		{
+			// Ignore Monolog log files (see #2579)
+			if (preg_match('/^[a-z]+-\d{4}-\d{2}-\d{2}\.log$/', $strFile))
```

Comment:
this wouldn't match `isotope_postsale-2020-01-01.log`. I think we should only match on the date at the end of the file name, wdyt?

Reply:
Changed in babdc77f550dd602153eb3e26ab7dcd03b2ee360.